py/client/pydeephaven/query.py

```diff
@@ -23,12 +23,13 @@ class Query(TableInterface):
     Note, an application should always use the factory method on the Session object to create a Query instance as the
     constructor is subject to future changes to support more advanced features already planned.
     """
+    _ops: List[TableOp]
```

Comment:
I see this as being added, but I don't see it being used anywhere in the diff I'm looking at.  Maybe I'm missing it in this view.

Reply:
It is used in a few places.